get_github_coderabbit_review/process_pr_comments.py

```
@@ -0,0 +1,255 @@
+""")
+            for file_path in sorted(files_mentioned):
+                file_comment_count = sum(1 for c in inline_comments if c.get('path') == file_path)
+                f.write(f"- `{file_path}`: {file_comment_count}件\n")
+
+            f.write(f"""
+## データファイル
+- **構造化データ**: `{output_file}`
+- **元データ**: `pr_104_raw_data.json`, `pr_104_inline_comments.json`, `pr_104_reviews.json`
+
+## AIエージェント向け情報
+このデータはAIエージェントが以下の用途で活用できます：
```

Ruff (0.12.2)

String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)